akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMultiMap.scala

```diff
@@ -241,6 +266,10 @@ final class ORMultiMap[A, B] private[akka] (
    * and another one is added within the same Update. The order of addition and removal is important in order
    * to retain history for replicated data.
    */
+  def replaceBinding(key: A, oldElement: B, newElement: B)(implicit node: SelfUniqueAddress): ORMultiMap[A, B] =
```

Comment:
does this work, I thought it would clash with the old since the first parameter list is the same?

Reply:
Hrm, all DD tests pass, did we find a compiler bug? I’ll check this.

Reply:
I'm pretty sure I saw that problem for the `+` operator, and thus the new name, but I could be wrong.

Reply:
It seems the compiler doesn't catch it unless it it used? Strange because I did see the compiler fail on ones using + or - vs a function name. Bug?